picamera2/previews/qt.py

```diff
@@ -1,2 +1,11 @@
-from .q_gl_picamera2 import EglState, QGlPicamera2
+# Installing Qt and OpenGL on the 64-bit Lite OS, and trying to run
+# a remote preview window, causes an error here (the 32-bit Lite OS is OK).
+# It may be something to do with more recent versions of python3-opengl?
+# Anyway, if we carry on regardless at least the non-OpenGL preview works,
+# which is in any case what is required for remote preview windows.
+try:
+    from .q_gl_picamera2 import EglState, QGlPicamera2
+except Exception:
+    print("Warning: OpenGL will not be available")
```

Comment:
Just wondering, should we use the logger here, rather than print, or is that not possible here?